### PR TITLE
Update support for GHC 9.0.1

### PR DESCRIPTION
**Pull request by CristhianMotoche**
## Added
* Support GHC 9.0.1

## Removed
* Support older versions of GHC (< 8.6)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## MASTER
 
 ## Dotenv 0.9.0.1
+### Added
```
Comment (fefi95) [inline CHANGELOG.md:4]: Should we add in here that the support for some 8.x versions was dropped and we added 9.0.1?

**Review by fefi95 — COMMENTED** [review]
Looking good! Thanks for working on this!

I left a comment, let me know if it's relevant 😃

File: CHANGELOG.md (hunk 1 above)
Comment (CristhianMotoche) [reply CHANGELOG.md:4]: Mmm that's a good one. Thanks!

**Comment by CristhianMotoche** [comment]
Thanks @fefi95 Let me know if you have any additional comments.

**Review by fefi95 — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**